README.md

```diff
@@ -5,6 +5,17 @@
 
 When the CRDs are present in the cluster, the operator can either be run outside or inside the cluster.
 
+## Prerequisites/Dependencies
+### podman
+This project relies strongly on Podman. It is used in the edge devices and also in the operator.
```

Comment:
Could you clarify what's the meaning of "podman...also in the operator" ?

Reply:
you and I know we do not use podman directly in the operator. at least not at the moment (see ). But operator does rely on poman version for validation. That' what I meant. If you have a different suggestion I'll be happy to use it. I was just trying to make sure users/developers have all the info.